projects/components/src/sequence/sequence-chart.component.ts

```diff
@@ -115,7 +115,6 @@ export class SequenceChartComponent implements OnChanges {
       id: segment.id,
       start: segment.start - minStart,
       end: segment.end - minStart,
-      label: segment.label,
```

Comment:
I couldn't see anywhere this was used, so removed it. cc: @anandtiwary to confirm this is okay.